desktop/src/ui/preload.ts

```diff
@@ -19,21 +19,27 @@ export function concatLink (host: string, path: string): string {
 
 async function loadServerConfig (url: string): Promise<any> {
   let retries = 5
-  let error: any
+  let res: Response | undefined
 
   do {
     try {
-      return await (await fetch(url)).json()
+      res = await fetch(url)
+      break
     } catch (e) {
```

Comment:
@haiodo now it only catches and retries `fetch` exceptions which are mostly consist of connectivity issues.